### PR TITLE
Add Mainnet Endpoints by NodeStake

### PR DESCRIPTION
**Pull request by nodestake**
## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: nodes/mainnet.md [hunk 1]
```diff
@@ -121,7 +124,7 @@ There are multiple explorers you can use for Mainnet Beta:
 - [https://celenium.io](https://celenium.io)
 - [https://celestiascan.com](https://celestiascan.com)
 - [https://mintscan.io/celestia](https://mintscan.io/celestia)
-- [https://celestia.explorers.guru](https://celestia.explorers.guru)
```
Comment (jcstein) [inline nodes/mainnet.md:124]: this is out of scope for this PR. why are you removing this explorer?

**Review by jcstein — CHANGES_REQUESTED** [review]

File: nodes/mainnet.md (hunk 1 above)
Comment (nodestake) [reply nodes/mainnet.md:124]: Oh, sorry, My mistake.
Fixed.

**Review by jcstein — APPROVED** [review]
:shipit:
